src/Halogen/VDom/Driver.purs

```diff
@@ -102,7 +100,11 @@ mkSpec handler renderChildRef document =
           ThunkSlot t -> do
             step' <- EFn.runEffectFn2 V.step step t
             pure $ V.mkStep $ V.Step (V.extract step') (Just step') patch done
-        _ -> EFn.runEffectFn1 render slot
+          PortalSlot elem inner -> do
+            EFn.runEffectFn1 V.halt step
+            EFn.runEffectFn2 renderPortalSlot elem inner
```

Comment:
I'm somewhat guessing about what to do here; this appears to work, but I don't know if I should instead be writing something closer to the `ThunkSlot` implementation.

Reply:
This is going to destroy the old DOM and build it from scratch on every render.